appium/webdriver/extensions/remote_fs.py

```diff
@@ -13,50 +13,54 @@
 # limitations under the License.
 
 import base64
+from typing import Optional, TypeVar
 
 from selenium import webdriver
 from selenium.common.exceptions import InvalidArgumentException
 
 from ..mobilecommand import MobileCommand as Command
 
+T = TypeVar('T', bound='RemoteFS')
```

Comment:
How about bounding all these basic types to AppiumWebDriver class? these mixins are anyway not used separately, but as a part of their "main" class